### DISCUSSION TITLE
Need bester git commit flow

### DISCUSSION THREAD

**Discussion by letieu** [CopilotC-Nvim/CopilotChat.nvim#205] [Ideas]
Current flow to slow:
- ask copilot chat commit message with `:CopilotChatCommit`
- copy message
- run `git add .`
- run `git commit -e`
- paste message
- quit

Any one has bester idea ?

**Comment by deathbeam** [comment]
You can just do git add . then git commit and inside of that buffer run :CopilotChatCommit and press Ctrl-Y to replace the buffer with the response which is much faster. Alternatively you can setup custom callback that calls git commit automatically after response is received

**Reply by letieu** [reply]
I will try
